gcs-app/gcs.py

```diff
@@ -0,0 +1,43 @@
+import logging, os, random, string, sys
+
+from google.cloud import storage
+
+class Gcs(object):
+    """
+    This class handles the logic for reading and writing to a GCS bucket.
+    """
+    def __init__(self):
+        # Set logging
+        logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+        # Setup GCS configs
+        self.storage_client = storage.Client()
+
+    def write_to_file(self, file_name, string_length):
```

Comment:
It is odd to have a function in a class that takes a `string_length` and then it generates a random string to upload. You either need to have a function called something like `write_random_string_to_file` or better yet, just have `upload_blob` take a function definition like: 
```python
def upload_blob(self, bucket_name, file_name, contents)
   ...
```
Then you actually have a useful function in a library instead of a misnamed function that is hard-coded to do something.  It would be left up to the caller of the function to generate the random string and call `upload_blob` with the random string.